clarifai/client/app.py

```diff
@@ -199,6 +203,34 @@ def delete_workflow(self, workflow_id: str) -> None:
       raise Exception(response.status)
     self.logger.info("\nWorkflow Deleted\n%s", response.status)
 
+  def image(self) -> Image:
+    """Returns an Image object.
+    Returns:
+        Image: An Image object.
+    """
+    return Image(self.user_id, self.id)
+
+  def video(self) -> Video:
```

Comment:
Yeah, will update and commit. also taking [out the **type** from inputs](https://github.com/Clarifai/clarifai-python/pull/144#discussion_r1297959470) to support multimodal inputs in almost all the functions..So that we can use the inputs class methods for any type, example,
```python
app = User(<>).app(<>)
input_obj = app.inputs()
input_proto = input_obj.get_input_from_url(input_id = 'demo', image_url='url', text_url = 'url')
```